gifSearch/ajax.js

```diff
@@ -0,0 +1,62 @@
+var ajax = {};
+ajax.x = function () {
+    if (typeof XMLHttpRequest !== 'undefined') {
+        return new XMLHttpRequest();
+    }
+    var versions = [
```

Comment:
Wanted to know why we are checking all the XMLHttp till XMLhttp 2? (I am not aware of this. Hence, asking)

Reply:
It was for IE support, now they're all obsolete.